packages/main/src/Table.ts

```diff
@@ -312,7 +334,7 @@ class Table extends UI5Element {
 	/**
 	 * Defines if the loading indicator should be shown.
 	 *
-	 * <b>Note:</b> When the component is loading, it is non-interactive.
+	 * **Note:** When the component is loading, it is non-interactive.
```

Comment:
might be easier to read: 
```suggestion
	 * **Note:** When the component is loading, it is not interactive.
```